test/pls.jl

```diff
@@ -132,7 +132,21 @@ end
     # since we're caching the fits, we should get it back to being correctly fitted
     # we also take this opportunity to test fitlog
     @testset "fitlog" begin
+        thin = 1
+        refit!(fm1; REML=false, progress=false, thin)
         fitlog = fm1.optsum.fitlog
+        fitlogtbl = columntable(fm1.optsum)
+        @test length(fitlogtbl) == 3
+        @test keys(fitlogtbl) == (:iter, :objective, :θ)
+        @test length(first(fitlogtbl)) > 15   # can't be sure of exact length
```

Comment:
since `thin=1`, this should be just `feval`

Reply:
We'll defer on this one a bit because presently it is `feval + 1`